### PR TITLE
Add see-also middleware

### PR DESCRIPTION
**Pull request by cskksc**
Before submitting a PR make sure the following things have been done:

- [ ] The commits are consistent with our [contribution guidelines](CONTRIBUTING.md)
- [x] You've added tests to cover your change(s)
- [x] All tests are passing
- [ ] The new code is not generating reflection warnings
- [x] You've updated the readme (if adding/changing middleware)

Thanks!

Given a ns and a symbol, return the list of related symbols. This will be used to show clojuredocs style "See Also" documentation links.
The clojuredocs see-also database is extracted out into a edn file and used to get this information.

### CODE REVIEW

**Comment by Malabarba** [comment]
As I said on the other PR, I think this information should be returned as an extra key in the info op. No reason it should have a separate middleware.

**Comment by cskksc** [comment]
Oh, right. Ill update both the PR's with all the changes. :-)

**Comment by cskksc** [comment]
Done. Some unrelated, `debug-integration-test` tests have failed.

**Comment by bbatsov** [comment]
I'd format the edn file properly. Right now it's just one huge line.

File: src/cider/nrepl/middleware/info.clj [hunk 1]
```diff
@@ -138,6 +138,9 @@
     (update-in info [:file] #(or (find-cljx-source %) %))
     info))
 
+(def see-also-data
+  (read-string (slurp "resources/see-also.edn")))
```
Comment (Malabarba) [inline src/cider/nrepl/middleware/info.clj]: Use `clojure.edn/read`.

**Comment by cskksc** [comment]
Changed the edn file to have 1 symbol per line. Also used `clojure.edn/read`.

**Comment by bbatsov** [comment]
👍

**Comment by cskksc** [comment]
🎉 I'll start the docstring parser today.

**Pull request merged**